scripts/install_xcpng.py

```diff
@@ -0,0 +1,219 @@
+#!/usr/bin/env python3
+import argparse
+import atexit
+import os
+import requests
+import sys
+import tempfile
+import time
+try:
+    sys.path.append(f"{os.path.abspath(os.path.dirname(__file__))}/..")
```

Comment:
Is this necessary? and why?

Reply:
In our tests yes as pytest handles the PYTHONPATH itself, but here unless we rework the structure entirely I don't see a better way.

Reply:
Just another question: Why this line is in a try/except block?

Reply:
it's the only way to pass pycodestyle.

Reply:
What's the warning? Because we have this piece of code in `runner.py` and it's fine:
```python
cur = os.path.split(__main__.__file__)[0]
root = os.path.abspath(os.path.join(cur, "..", ".."))
sys.path.append(root)
```

Reply:
I just remembered a better way to silence the warning:

```
sys.path.append(f"{os.path.abspath(os.path.dirname(__file__))}/..") # noqa
```